app/routes/admin/groups/components/GroupMembersList.tsx

```diff
@@ -71,8 +77,63 @@ const GroupMembersList = ({
     </Link>
   );
 
-  const RoleRender = (role: string) =>
-    role !== 'member' && <i>{ROLES[role] || role} </i>;
+  const RoleRender = (fullName, membership: Membership) => {
```

Comment:
Does this work at all? The `dataIndex` for this is `role` and you just randomly pass it a fullName and Membership
<img width="581" alt="image" src="https://user-images.githubusercontent.com/23152018/230774730-966e8255-fe07-4312-90dc-03ba35b6045f.png">

Reply:
Again, no idea how `dataIndex` actually works. It just works 🤷🏼 I removed `dataIndex: 'edit'` though.

Reply:
It is only used for the filtering btw.

Reply:
Me neither 🤷🏻 🙃 But I remember now. The `dataIndex` is used to basically keep track of everything for that column. Basically whatever you do on that column will only work if the `dataIndex` is set.

<img width="350" alt="image" src="https://user-images.githubusercontent.com/23152018/230775764-2243b170-69a2-4b5f-b3de-eafa232856a1.png">